### PR TITLE
add comments to :move-to function

### PR DESCRIPTION
**Pull request by k-okada**

### CODE REVIEW

File: pr2eus/robot-interface.l [hunk 1]
```diff
@@ -1354,6 +1354,9 @@ Return value is a list of interpolatingp for all controllers, so (null (some #'i
   ;;
   (:move-to
    (coords &rest args &key (no-wait nil) &allow-other-keys)
+   "move the robot to coordinates in reference defined in :frame-id keyward. The default frame is \"world\"
+    Return t if th if the robot reached the goal. By defaut, it blocks until the robot reached the goal.
```
Comment (knorth55) [inline pr2eus/robot-interface.l:1358]:
```suggestion
    Return t if the robot reached the goal. By defaut, it blocks until the robot reached the goal.
```

**Review by knorth55 — CHANGES_REQUESTED** [review]

File: pr2eus/robot-interface.l (hunk 1 above)
Comment (708yamaguchi) [inline pr2eus/robot-interface.l:1358]: By defaut
->
By default

**Review by 708yamaguchi — CHANGES_REQUESTED** [review]

File: pr2eus/robot-interface.l [hunk 2]
```diff
@@ -1354,6 +1354,9 @@ Return value is a list of interpolatingp for all controllers, so (null (some #'i
   ;;
   (:move-to
    (coords &rest args &key (no-wait nil) &allow-other-keys)
+   "move the robot to coordinates in reference defined in :frame-id keyward. The default frame is \"world\"
```
Comment (mqcmd196) [inline pr2eus/robot-interface.l:1357]:
```suggestion
   "move the robot to coordinates in reference defined in :frame-id keyword. The default frame is \"world\"
```

**Comment by k-okada** [comment]
thank you, fixed comment . Should we add documen tot `:move-to-wait`? I am sure that `:move-to-send`is private method, so we do not need documentation. How about `:move-to-wait` ? Is it also private method?

**Comment by 708yamaguchi** [comment]
I do not use `:move-to-wait` explicitly in a program, so in my opinion, `:move-to-wait` is a private method.

**Pull request merged**